jdi-bdd/pom.xml

```diff
@@ -18,17 +18,7 @@
         <dependency>
             <groupId>com.epam.jdi</groupId>
             <artifactId>jdi-light-html</artifactId>
-            <version>1.2.0</version>
```

Comment:
cucumber testng - may be we still need it

Reply:
Change from master, seems that we don't need it

Reply:
Reverted